app/players/dwarves/dwarf_blacksmith.py

```diff
@@ -0,0 +1,16 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfBlacksmith(Dwarf):
+    def __init__(
+            self, nickname: str, favourite_dish: str, skill_level: int
+    ) -> None:
+        super().__init__(nickname, favourite_dish)
+        self._skill_level = skill_level
+
+    def player_info(self) -> str:
+        return f"Dwarf blacksmith {self.nickname} " \
```

Comment:
I'm pretty sure we don't use `\` as line break here. Use parentheses instead:
```python
return (f"Dwarf blacksmith {self.nickname} "
        f"with skill of the {self._skill_level} level")
```

Reply:
I'm not sure which is the best practice, so I want to leave it like it is

Reply:
I agree, using `( )` is better